.gitignore

```diff
@@ -6,3 +6,5 @@
 /src/contracts
 /src/features
 /src/json
+/terminus-test-resources
```

Comment:
Please remove /terminus-test-resources you need to commit in this folder if you make changes to the sub-module